src/components/deposit/Confirming.js

```diff
@@ -48,28 +36,22 @@ const Confirming = ({
             {signerFee} BTC*
           </div>
         </div>
+        <div className="error">
+          { error }
+        </div>
```

Comment:
Not blocking, but it feels like it would make more sense to swap this with the description, rather than have both visible when there's an error.